docbrown/core/src/graph.rs

```diff
@@ -564,6 +579,14 @@ impl<'a> VertexView<'a, TemporalGraph> {
     pub fn global_id(&self) -> u64 {
         self.g_id
     }
+    
+    pub fn partition_id(&self) -> usize {
+        self.pid
+    }
+
+    pub fn window(&self) -> Option<Range<i64>> {
+        self.w.clone()
```

Comment:
do we really need to clone this?

Reply:
yup it doesnt implement the copy trait so we have to use clone